schemas/schemas/infoQuestion.json

```diff
@@ -227,7 +227,16 @@
         "workspaceOptions": {
             "description": "Options for workspace questions.",
             "type": "object",
-            "additionalProperties": false,
+            "additionalProperties": {
+                "syncIgnore": {
+                    "description": "(deprecated) The list of files or directories that will be excluded from sync.",
+                    "type": "array",
+                    "items": {
+                        "description": "A single file or directory that will be excluded from sync.",
+                        "type": "string"
+                    }
+                }
```

Comment:
I moved the property here so that it wouldn't produce a sync error. Is it work emitting a sync warning for this? I'm erring towards no, as it could be quite noisy for courses.

Reply:
I'd also suggest no warning. I believe a lot of instructors may have added this to their VSCode-based workspaces based on the existing examples in docs and the example code.